mne_sandbox/preprocessing/bads/tests/test_bads.py

```diff
@@ -0,0 +1,239 @@
+import numpy as np
+from nose.tools import (assert_true, assert_raises, assert_equal)
+from numpy.testing import (assert_allclose, assert_almost_equal)
+import mne
+
+from mne_sandbox.preprocessing.bads.faster_ import (_hurst, _freqs_power)
+from mne_sandbox.preprocessing.bads import (find_bad_channels, find_bad_epochs,
+                                            find_bad_channels_in_epochs)
+
+# Signal properties used in the tests
+length = 2  # in seconds
+srate = 200.  # in Hertz
+n_channels = 32
+n_epochs = 100
+n_samples = int(length * srate)
+time = np.arange(n_samples) / srate
+
+# Fix the seed
+np.random.seed(123)
+
+
+def test_hurst():
+    """Test internal hurst exponent function."""
+    np.random.seed(123)
```

Comment:
usually we do `rng = np.random.RandomState(123)` and use methods instead of messing with global states